packages/app-bridge/src/types/Guideline.ts

```diff
@@ -78,3 +78,12 @@ export interface GuidelineDocumentPageHeading {
     slug(language?: string): string;
     type: 'document-page-heading';
 }
+
+export type PortalNavigationItem =
+    | GuidelineCoverPage
+    | GuidelineDocumentGroup
+    | GuidelineDocument
+    | GuidelineDocumentLibrary
+    | GuidelineDocumentLink;
+
+export type PortalNavigationItemTypes = PortalNavigationItem['type'];
```

Comment:
It seems sub-optimal to have two representations of the same type. Let's see how this develops.